src/types.js

```diff
@@ -5,6 +5,11 @@ export type PackageAtVersion = {
   version: string
 };
 
+export type Node = PackageAtVersion & {
```

Comment:
Doesn't look like this type is being used?

Reply:
I think I have another PR ready that does use it

Reply:
👍